src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -117,6 +117,40 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AssemblyInitializeShouldBeValidDescription" xml:space="preserve">
+    <value>Assembly Initialize attribute should follow the following layout to be valid:
```

Comment:
```suggestion
    <value>Assembly Initialize methods should follow the following layout to be valid:
```

Reply:
or maybe even better as 

```suggestion
    <value>Methods marked with [AssemblyInitialize] should follow the following layout to be valid:
```

Reply:
Oooh and I just noticed but `should follow the following` doesn't read so well. It's possible I didn't notice on the previous PRs... sorry.

What about `should adhere to the following layout`?

Reply:
@nohwnd I need some English writting check, my brain is confused.

Reply:
LGTM